android_mobile/src/main/java/com/alexstyl/specialdates/donate/DonateActivity.java

```diff
@@ -81,13 +83,13 @@ public void onClick(View v) {
             @Override
             public void run() {
                 scrollToDonate();
-
             }
 
             private void scrollToDonate() {
                 CoordinatorLayout.LayoutParams params = (CoordinatorLayout.LayoutParams) appBarLayout.getLayoutParams();
                 AppBarLayout.Behavior behavior = (AppBarLayout.Behavior) params.getBehavior();
-                behavior.onNestedFling(coordinator, appBarLayout, null, 0, 50, true);
+                if (behavior != null)
```

Comment:
If you run the app on an emulator like I did, donation activity will crash throwing NPE due to behavior being null so I put this null check to at least prevent the app from crashing even though donation activity won't open on an emulator anyways. Let me know how should I proceed with this.
![behaviornull](https://cloud.githubusercontent.com/assets/4888330/25785676/74faac10-338f-11e7-95a5-280e3ac6fc2e.png)